### DISCUSSION TITLE
How to: Make KDE applications look good and work fine on Hyprland

### DISCUSSION THREAD

**Discussion by Aqa-Ib** [hyprwm/Hyprland#5030] [Show and tell]
This package https://aur.archlinux.org/packages/qt6ct-kde is essential for KDE programs like Dolphin to look good and work properly on Hyprland. Couple it with `env = QT_QPA_PLATFORMTHEME,qt6ct` in Hyprland config. Enjoy it !

**Comment by vaxerski** [comment]
![image](https://github.com/hyprwm/Hyprland/assets/43317083/d5767cc3-8c17-43b0-9e06-fe8ef2f973f3)

**Reply by Aqa-Ib** [reply]
True, kvantum works too. However I think that this package is superior bacause it allows you to change color schemes instantly without restarting the application, and also it supports 2000 KDE colorschemes available at https://store.kde.org/browse?cat=112&ord=latest

**Reply by vaxerski** [reply]
fair enough, but I like my kvantum

**Comment by spikespaz** [comment]
I could never get https://github.com/PapirusDevelopmentTeam/materia-kde to work with `qt6ct` alone. I tried, because I think that `materia-kde-theme` -> `kvantum` -> `qt6ct`  causes some breakages in the theme.
